src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/Licenses.arcade.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Should we consider having sub-folders for various test-specific baselines, now that we have significantly more items in the main `baselines` folder? I.e. `baselines/licenses`

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/21d4b38506b734d9facc797f92c069bbaa8caefd